src/modules/Profile/Profile.tsx

```diff
@@ -81,8 +83,18 @@ type ProfileProps = {
 };
 
 function Profile({ user, action }: ProfileProps): JSX.Element {
-  const handleAvatarChange = useCallback((e) => {
-    alert(`Загрузить ${e.target.files[0].name}`);
+  const dispatch = useAppDispatch();
+  const handleAvatarChange = useCallback(async (e) => {
```

Comment:
🧐
Вкусовщина, я люблю отделять логические блоки пустой строкой. Сначала идёт кастомный хук, потом коллбэк, такое можно было бы разделить. Опять же, вкусовщина, если согласна, то поправь, если нет, то и так хорошо)
```suggestion
  const dispatch = useAppDispatch();
  
  const handleAvatarChange = useCallback(async (e) => {
```